src/main/java/dk/sdu/se_f22/contentmodule/index/DB/Database.java

```diff
@@ -0,0 +1,167 @@
+package dk.sdu.se_f22.contentmodule.index.DB;
+
+import java.sql.*;
+import java.util.ArrayList;
+
+public class Database {
```

Comment:
I would assume the shared connection should be used instead of individual implementations.